src/components/Modals/UploadDocumentModal.jsx

```diff
@@ -17,19 +16,23 @@ import { DatePicker } from '@mui/x-date-pickers/DatePicker';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import useMediaQuery from '@mui/material/useMediaQuery';
 import { useTheme } from '@mui/material/styles';
-/* import Slide from '@mui/material/Slide';
- */ // Context Imports
+/* import Slide from '@mui/material/Slide'; */
```

Comment:
Might want to use it in the future if we want animations further down the line, but for now, think we could remove this import.